### PR TITLE
Update black with click dependencies

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

File: test-lib.sh [hunk 1]
```diff
@@ -36,7 +36,8 @@ function ct_cleanup() {
     docker stop "$container"
 
     # Container has not been removed by `docker stop` and still exists
-    if [ $( docker ps -a -f id=$container | wc -l ) -eq 2 ]; then
+    # shellcheck disable=SC2046
```
Comment (pkubatrh) [inline test-lib.sh:39]: Why are we disabling the check instead of using double quotes as expected?

Comment (phracek) [reply test-lib.sh:39]: I don't like double quotes twice. :( But I can change it.

Comment (phracek) [reply test-lib.sh]: Like here: https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L79

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
The Sanity tests are passing. This does not affect tests. Merging it.

**Pull request merged**